lib/src/DateRangePicker/DateRangePickerToolbar.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import PickerToolbar from '../_shared/PickerToolbar';
+import Typography from '@material-ui/core/Typography';
+import { useUtils } from '../_shared/hooks/useUtils';
+import { makeStyles } from '@material-ui/core/styles';
+import { ToolbarComponentProps } from '../Picker/Picker';
+import { ToolbarButton } from '../_shared/ToolbarButton';
+import { DateRange, CurrentlySelectingRangeEndProps } from './RangeTypes';
+
+export const useStyles = makeStyles(
+  {
+    penIcon: {
+      position: 'relative',
+      top: 4,
+    },
+    dateTextContainer: {
+      display: 'flex',
+    },
+  },
+  { name: 'MuiPickersDatePickerRoot' }
+);
+
+interface DateRangePickerToolbarProps
+  extends CurrentlySelectingRangeEndProps,
+    Pick<
+      ToolbarComponentProps,
+      'isMobileKeyboardViewOpen' | 'toggleMobileKeyboardView' | 'toolbarTitle' | 'toolbarFormat'
+    > {
+  date: DateRange;
+  startText: React.ReactNode;
+  endText: React.ReactNode;
+  currentlySelectingRangeEnd: 'start' | 'end';
+  setCurrentlySelectingRangeEnd: (newSelectingEnd: 'start' | 'end') => void;
+}
+
+export const DateRangePickerToolbar: React.FC<DateRangePickerToolbarProps> = ({
+  date: [start, end],
+  toolbarFormat,
+  isMobileKeyboardViewOpen,
+  toggleMobileKeyboardView,
+  currentlySelectingRangeEnd,
+  setCurrentlySelectingRangeEnd,
+  startText,
+  endText,
+  toolbarTitle = 'SELECT DATE RANGE',
+}) => {
+  const utils = useUtils();
+  const classes = useStyles();
+
+  const startDateValue = start
+    ? utils.formatByString(start, toolbarFormat || utils.formats.shortDate)
+    : startText;
+
+  const endDateValue = end
+    ? utils.formatByString(end, toolbarFormat || utils.formats.shortDate)
+    : endText;
+
+  return (
+    <PickerToolbar
+      toolbarTitle={toolbarTitle}
+      isMobileKeyboardViewOpen={isMobileKeyboardViewOpen}
+      toggleMobileKeyboardView={toggleMobileKeyboardView}
+      isLandscape={false}
+      penIconClassName={classes.penIcon}
+    >
+      <div className={classes.dateTextContainer}>
+        <ToolbarButton
+          variant={Boolean(start) ? 'h5' : 'h6'}
```

Comment:
Do we need to cast the variable to a boolean?

```suggestion
          variant={start ? 'h5' : 'h6'}
```

Reply:
My point: implicit types castings are the root of evil

Reply:
It seems simple enough here 🤔, what would be the downside?

Reply:
Maybe it is simple here. But what if an empty string will be able to pass to the ‘start’?
This approach makes it easier to understand that you are doing unsafe in JavaScript operation. 

And again in strongly typed languages like Haskell, OCaml, Rust, C# and many other. It’s just a story about strictness. 

But only one thing I’m worrying about — bundlesize :(

Reply:
What's the approach used in the main repository? If it already works well for us, I would stick to it.

Reply:
Yeah, it is an unpopular way to handle things. But it doesn't mean that everybody can't be wrong 
![image](https://user-images.githubusercontent.com/16926049/78880202-b8407280-7a5d-11ea-867e-455820e99b0a.png)